flask_mongorest/views.py

```diff
@@ -28,7 +28,7 @@ def serialize_mongoengine_validation_error(e):
 
     def serialize_errors(e):
         if getattr(e, 'message', None):
```

Comment:
Don't think this makes sense in its current form – we check for the existence of `e.message` and then return `e.args[0]`?